bindings/go/mio/mio.go

```diff
@@ -186,21 +189,13 @@ func (mio *Mio) objNew(id string) (err error) {
     return nil
 }
 
-func (mio *Mio) finishOpen(sz uint64) {
-    mio.buf = make([]C.struct_m0_bufvec, threadsN)
-    mio.ext = make([]C.struct_m0_indexvec, threadsN)
-    mio.attr = make([]C.struct_m0_bufvec, threadsN)
-    mio.ch = make(chan slot, threadsN)
-    // fill the pool with slots
-    for i := 0; i < threadsN; i++ {
-        mio.ch <- slot{i, nil}
-    }
+func (mio *Mio) open(sz uint64) {
```

Comment:
Please document this function as an internal function.
It can not, should not be used directly by user.
Can go limit the direct use of a function?

Not export?
Private?

Reply:
@huanghua78 all functions in Go which start with lowercase letter are internal and not exported. This is not needed to be documented.